haystack/components/converters/docx.py

```diff
@@ -79,11 +82,16 @@ class DOCXToDocument:
     ```
     """
 
-    def __init__(self):
+    def __init__(self, table_format: Literal["markdown", "csv"] = "markdown"):
```

Comment:
Let's expose this as an enum like we do with other components.

Reply:
Ok deal, will do 🙏